src/types/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './contacts.types';
+export * from './store.types';
+
+export type ErrorType = {message?: any};
+export type ChildrenType = {children?: any};
```

Comment:
instead of any for now we can specify `React.ReactNode`. 
Deatils: https://stackoverflow.com/q/53688899